utilities/src/fixed128.rs

```diff
@@ -7,12 +7,12 @@ use sp_runtime::{
 };
 
 #[cfg(feature = "std")]
-use serde::{Deserialize, Serialize};
+use serde::{Deserialize, Serialize, Serializer};
 
 /// An unsigned fixed point number. Can hold any value in the range [0, 340_282_366_920_938_463_464]
 /// with fixed point accuracy of 10 ** 18.
 #[derive(Encode, Decode, Default, Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "std", derive(Deserialize))]
```

Comment:
Serialize and Deserialize won't match like this and it could cause issue on chain spec generation

Reply:
Added `Deserialize` impl.